documentation/book/proc-exposing-kafka-bridge-service-local-machine.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-quickstart.adoc
+
+[id='proc-exposing-kafka-bridge-service-local-machine-{context}']
+= Exposing the Kafka Bridge Service to your local machine
+
+Next, create an OpenShift Route to expose the {ProductName} Kafka Bridge Service to your local machine on port 8080.
```

Comment:
I think this makes it too OpenShift specific. We should use `port-forward` or `proxy` if we want something working everywhere - that should work fine for some quickstart. If this should be more production like, we should cover Kubernetes options - Ingress, LoadBalancers, NodePorts.

Reply:
Agree. We cannot use an OpenShfit route even because the user could use minikube (so Kubernetes, so no routes).
For a quickstart I would go for just `port-forward`. We can add usage of Ingress and more with a blog post or a more advanced documentation section.

Reply:
Agreed. I'll rewrite to use port forwarding.